pkg/upgrade/initialize.go

```diff
@@ -10,28 +10,31 @@ import (
 	"github.com/openshift/openshift-azure/pkg/util/azureclient/storage"
 )
 
-func (si *simpleUpgrader) InitializeCluster(ctx context.Context, cs *api.OpenShiftManagedCluster) error {
-	authorizer, err := azureclient.NewAuthorizerFromContext(ctx)
-	if err != nil {
-		return err
+func (u *simpleUpgrader) InitializeCluster(ctx context.Context, cs *api.OpenShiftManagedCluster) error {
+	if u.accountsClient == nil {
```

Comment:
this can move to NewSimpleUpgrader, can't it?

Reply:
it uses the context and we don't have that in the NewSimpleUpgrader

Reply:
@amanohar or @sozercan is the Context object containing the Azure credentials (api.ContextKeyClientID etc) available when the RP calls plugin.NewPlugin?

Reply:
opened follow up issue #602